site/app/templates/banner/Banner.twig

```diff
@@ -0,0 +1,35 @@
+<div class="content">
+
+<button class="Upload_Banner" onclick="newUploadBanner()"> Upload a Banner </button>
```

Comment:
![image](https://github.com/Submitty/Submitty/assets/47994802/8a8e59c2-924e-4a56-8b35-298425c36c42)
Class should probably also have btn and btn-primary so that the CSS works properly